wagtailio/navigation/models.py

```diff
@@ -39,6 +46,13 @@ class MainMenu(ClusterableModel):
         FieldPanel("menu_sections", classname="collapsible"),
     ]
 
+    def save(self, **kwargs):
+        super().save(**kwargs)
+
+        if NavigationSettings.objects.filter(main_navigation=self).exists():
+            print("delete")
```

Comment:
Issue: Print

Reply:
poor man's debugger 😂
(context: PyCharm 2022.2 foobared Docker support in some interesting ways)